CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Change Log
 
-## Unreleased
+## 22.8.1
 
-* Add support for flake8 5.0.0.
+* Add support for flake8 >= 5.0.0.
```

Comment:
Not 100% correct, since we don't support 6+, but good enough.

Reply:
'Tis but a changelog, not a legal thesis